meshdb/meshdb.go

```diff
@@ -568,6 +568,19 @@ func parseContractAddressesAndTokenIdsFromAssetData(assetData []byte, contractAd
 			}
 			singleAssetDatas = append(singleAssetDatas, a)
 		}
+	case "StaticCall":
+		var decodedAssetData zeroex.StaticCallAssetData
+		err := assetDataDecoder.Decode(assetData, &decodedAssetData)
+		if err != nil {
+			return nil, err
+		}
+		// NOTE(jalextowle): As of right now, none of the supported staticcalls
+		// have important information in the StaticCallData. This will probably
+		// change, which may require addtional information from this case.
+		a := singleAssetData{
```

Comment:
Now that I look at this again, there really isn't a reason to add this for the current staticcalls. We aren't watching any events from that address, so this will just waste space. It also doesn't really make sense in the context of the callbacks registered in the `setupOrders` function. This may change with future staticcalls. 

I'd be in favor of fixing this before the release (I'll make a PR after this review).